### DISCUSSION TITLE
classes/class_gpuparticlesattractorvectorfield3d

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#30] [User-contributed notes]
# classes/class_gpuparticlesattractorvectorfield3d

Inherits: GPUParticlesAttractor3D< VisualInstance3D< Node3D< Node< Object A box-shaped attractor with varying directions and strengths defined in it that influences particles from GPUParticles3D no...

https://docs.godotengine.org/en/latest/classes/class_gpuparticlesattractorvectorfield3d.html

**Comment by ZenithStar** [comment]
Note that the texture is centered at (0.5f, 0.5f, 0.5f) #808080 for zero acceleration.

**Reply by Calinou** [reply]
Thanks for bringing it up! I opened a PR to document this: https://github.com/godotengine/godot/pull/93414

I did my own testing to confirm how it works too.
